eidas-saml-parser/src/dist/config.yml

```diff
@@ -30,3 +30,5 @@ replayChecker:
   redisUrl: ${REDIS_SERVER_URI:-https://DefaultNotUsed}
 
 proxyNodeAuthnRequestUrl: ${PROXY_NODE_AUTHN_REQUEST_ENDPOINT:-DefaultNotUsed}
+
+prometheusEnabled: ${PROMETHEUS_ENABLED:-true}
```

Comment:
do you anticipate ever switching this off? If not I'd hardcode it to `true` in code. hot take: configuration options are tech debt